src/components/GetStarted/EditPreferences.vue

```diff
@@ -0,0 +1,295 @@
+<template>
+	<div class="edit-preferences">
+		<h2 class="edit-preferences__header">
+			Looking for different loans?
+		</h2>
+		<p class="edit-preferences__sub-header">
+			Change your preferences and find more borrowers to support.
+		</p>
+		<div class="row">
+			<div class="small-12 large-6 columns">
+				<!-- Causes Card -->
+				<div class="edit-preferences__card" v-if="causes.length > 0">
+					<div class="edit-preferences__icon-wrapper">
+						<span class="edit-preferences__icon-background">
+							<kv-icon
+								class="edit-preferences__check-icon"
+								name="checkmark"
+							/>
+						</span>
+					</div>
+					<div class="edit-preferences__content-wrapper">
+						<div class="row collapse text-left align-middle">
+							<div class="small-6 large-7 xlarge-6 columns">
+								<h2 class="edit-preferences__content-wrapper--title">
+									Your {{ causes.length }} causes
+								</h2>
+								<router-link
+									class="smallest"
+									to="/get-started"
+								>
+									Change causes &#8594;
+								</router-link>
+							</div>
+							<div class="small-6 large-5 xlarge-6 columns">
+								<ul class="edit-preferences__list">
+									<li
+										class="edit-preferences__list-item"
+										v-for="(cause, index) in causesTrimmed"
+										:key="index"
+									>
+										<kv-cause-selector
+											:cause="cause.name"
+											:as-icon="true"
+										/>
+									</li>
+									<li v-if="causes.length > 3" class="edit-preferences__list-item--counter">
```

Comment:
Same BEM comment as above

Reply:
I think this one is ok, it is a modifier on the list item.

Reply:
OK gotcha. Modifier classes are also always in addition to the normal class. So it'd end up being 
`class="edit-preferences__list-item edit-preferences__list-item--counter"`
Where your --counter CSS would be anything that overrides or is in addition to list-item.

It does feel a little annoying to double up on the classes. But also kinda nice once you're used to it.
The "Under the Hood" section shows a good example: http://getbem.com/introduction/